### DISCUSSION TITLE
ClickAsync(x,y)

### DISCUSSION THREAD

**Discussion by Mephisto090** [hardkoded/puppeteer-sharp#1986] [Ideas]
Hello, :)

The `ClickAsync` function is currently only available in such a way that it always presses in the middle of the button. Would it be possible to extend the function so that I can pass a Point (x,y) and press it there. If nothing was passed then in the middle?

`ClickAsync(x,y)`

**Comment by kblok** [comment]
If you want to pass an x/y coordinate you can use `Page.Mouse` which is what `ClickAsync` uses internally.
